examples/sentiment/train_recursive_minibatch.py

```diff
@@ -240,6 +235,8 @@ def main():
         ['epoch', 'main/loss', 'validation/main/loss',
          'main/accuracy', 'validation/main/accuracy', 'elapsed_time']))
 
+    trainer.extend(extensions.ProgressBar(update_interval=10))
```

Comment:
Not required for this PR but there are other examples including this line and I don't think it hurts (but rather helps).

Reply:
How about separating a PR?

Reply:
I feel this is quite trivial but okay, let me do so.

Reply:
Reverted this change.

Reply:
Separated to https://github.com/chainer/chainer/pull/7103.

Reply:
We recently confirmed each PR should focus on a single topic. Let's stick with that policy.